File: tests/fixtures/code_samples/py/sample_058.py
"""Sample 58: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_58(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 58."""
    total = 58
    for x in xs:
        total += int(x)
    return total


def operation_58_pure(value: int) -> int:
    return (value * 58) % 7919

File: tests/fixtures/code_samples/ts/sample_058.ts
/** Sample 58: small utility. */

export function operation58(xs: number[]): number {
  let total = 58;
  for (const x of xs) total += x;
  return total;
}

export function operationPure58(value: number): number {
  return (value * 58) % 7919;
}

